site/public/js/manage-sessions.js

```diff
@@ -0,0 +1,139 @@
+function removeRowTrigger(elem) {
+    const tr_elem = elem.parent().parent();
+    const table_elem = tr_elem.parent();
+    tr_elem.remove();
+    table_elem.trigger('change');
+}
+
+// eslint-disable-next-line no-unused-vars
```

Comment:
It would be better to add an ignore line to the top of the file.  See [here](https://submitty.org/developer/software_and_system_design/coding_style_guide/javascript#style-rules) for more information.  The same comment applies elsewhere throughout this file.

Reply:
Regular POST requests are sent now instead of AJAX so this file has been removed.